common/locales/en/settings.json

```diff
@@ -104,5 +104,7 @@
     "benefits": "Benefits",
     "coupon": "Coupon",
     "couponPlaceholder": "Enter Coupon Code",
-    "couponText": "We sometimes have events and give out coupon codes for special gear. (eg, those who stop by our Wondercon booth)"
+    "couponText": "We sometimes have events and give out coupon codes for special gear. (eg, those who stop by our Wondercon booth)",
+    "currentTasks": "To-do list",
+    "exportAllActiveTodos": "All my to-dos:"
```

Comment:
"To-Do" is the correct capitalisation (just for the text portion, not for the key).